examples/script-tag-dynamic-imports/astro.config.mjs

```diff
@@ -0,0 +1,4 @@
+import { defineConfig } from 'astro/config';
+
+// https://astro.build/config
+export default defineConfig({});
```

Comment:
Question: Why is there a need for an empty object here?

Reply:
another default from create astro 🤷